src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/mgmt_iot_hub/swagger_iot_hub_resource_provider.json

```diff
@@ -0,0 +1,1913 @@
+{
```

Comment:
This swagger is copied from https://github.com/Azure/azure-rest-api-specs/blob/master/arm-iothub/2016-02-03/swagger/iothub.json with some minor modifications. We made those modifications to make the auto-generated service client to work. Looks like the Swagger file has been updated recently. Will merge changes from there.